naga/Cargo.toml

```diff
@@ -15,7 +15,7 @@ autotests = false
 # copy the crates it actually uses out of the workspace, so it's meaningful for
 # them to have less restrictive MSRVs individually than the workspace as a
 # whole, if their code permits. See `../README.md` for details.
-rust-version = "1.76"
+rust-version = "1.80.0"
```

Comment:
Typo? Shouldn't this be 1.82?

Reply:
Ah, almost certainly. 👍🏻 Will fix.

Reply:
I have several commits in this PR, and I'd correctly adjust `rust-version` for the 1.76 → 1.80 bump, but not the subsequent bump to 1.82. 😅 Fixed now, thanks!